flink-streaming-java/src/test/java/org/apache/flink/streaming/api/operators/source/SourceOperatorEventTimeTest.java

```diff
@@ -161,6 +199,7 @@ public void testPerSplitOutputEventWatermarks() throws Exception {
 	@SuppressWarnings("FinalPrivateMethod")
 	@SafeVarargs
 	private final List<Object> testSequenceOfEvents(
+			SourceOperator.RuntimeMode runtimeMode,
```

Comment:
Here we use `final` for some args and nothing for others. So feel free to uniformize them the way you feel more comfortable.

Reply:
👌